flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -613,25 +614,31 @@ protected void cleanUpInvoke() throws Exception {
 		// stop all timers and threads
 		tryShutdownTimerService();
```

Comment:
This call has the same try-catch-log inside. Can we apply the same approach to it?

Reply:
I have no idea, the current change does not change the behavior of `cleanUpInvoke`. 

I am a bit hesitant to make changes to change the behavior.

Reply:
I don't mean to change behavior. Only to pull `try-catch` level up as it is done for other parts.